src/chaat/handler/handler.clj

```diff
@@ -0,0 +1,52 @@
+(ns chaat.handler.handler
+  (:require [ring.util.response :as res]
+            [chaat.model.user :as model.user]
```

Comment:
[non-blocking, suggestion] Can have this aliased to just `user` to make references to this more concise.

Reply:
I would like that as well because it is more concise. I was confused about the necessity to keep the distinction between namespaces because`model.user` and `db.user` both have `... .user`. But this distinction might not need to be explicit.
I will make this more concise in a future PR.